integrations/node/workload-executor.js

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env node
+
+const MongoClient = require('../../node-mongodb-native').MongoClient;
+const fs = require('fs');
+const assert = require('assert');
+const omit = require('lodash.omit');
+
+let interrupted = false;
+process.on('SIGINT', () => (interrupted = true));
+
+const executors = {
```

Comment:
two nits:
- `executors` => `operations`, these are definitions of individual operations that an executor can run
- I think using a `Map` would be preferable for readability (`if (operations.has(operationName)) { ... }`)